util/ranger/ranger_test.go

```diff
@@ -2097,3 +2098,91 @@ func TestShardIndexFuncSuites(t *testing.T) {
 		require.Equal(t, fmt.Sprintf("%s", newConds), tt.outputConds)
 	}
 }
+
+func getSelectionFromQuery(t *testing.T, sctx sessionctx.Context, sql string) *plannercore.LogicalSelection {
+	ctx := context.Background()
+	stmts, err := session.Parse(sctx, sql)
+	require.NoError(t, err)
+	require.Len(t, stmts, 1)
+	ret := &plannercore.PreprocessorReturn{}
+	err = plannercore.Preprocess(sctx, stmts[0], plannercore.WithPreprocessorReturn(ret))
+	require.NoError(t, err)
+	p, _, err := plannercore.BuildLogicalPlanForTest(ctx, sctx, stmts[0], ret.InfoSchema)
+	require.NoError(t, err)
+	selection := p.(plannercore.LogicalPlan).Children()[0].(*plannercore.LogicalSelection)
+	require.NotNil(t, selection)
```

Comment:
```suggestion
_, isSelection := p.(plannercore.LogicalPlan).Children()[0].(*plannercore.LogicalSelection)
	require.True(t, isSelection)
```

Reply:
We need `selection` in subsequent test. So I change it to 
```
selection, isSelection := p.(plannercore.LogicalPlan).Children()[0].(*plannercore.LogicalSelection)
require.True(t, isSelection)
```